tools/dbgen.py

```diff
@@ -398,6 +430,11 @@ def generate_object(obj: Object) -> CodeLines:
     lines = CodeLines()
     lines.header = [
         '',
+        '/**',
+        ' *',
+        f' *  @class ConfigDB class for {obj.typename}, generated from schema',
+        ' *',
+        ' */',
```

Comment:
There is a file header which says it's auto-generated, so does this really contribute anything useful? The generated code should be largely self-documenting; if it's not clear we change the code.